tools/linter_lib/custom_check.py

```diff
@@ -539,6 +539,8 @@
     {"pattern": "Terms of service", "description": "The S in Terms of Service is capitalized"},
     {
         "pattern": "[^-_p]botserver(?!rc)|bot server",
+        # api-release-checklist refers to the API packages by name
+        "exclude": {"docs/subsystems/api-release-checklist.md"},
```

Comment:
I don't understand why we need this exclude -- the the `pattern` has a negative lookahead for `-` or `_` appearing before the `botserver` token.